### ISSUE TITLE
Implement passphrase generation

### ISSUE THREAD

**Issue by agyild** [pwsafe/pwsafe#564]
Entropy-wise passphrases are more secure compared to memorable passwords as explained [here](https://www.useapassphrase.com/). That is why a passphrase generator is a necessary feature.

**Comment by ronys** [comment]
- To do this properly we need to support non-English passphrases as well.
- I'm not sure the added size of dictionaries to the PasswordSafe distributable is worth the benefit
- A long enough random password string can be as strong as the user wishes, since PasswordSafe's whole purpose is to free the user from the need to remember these...
- Finally, there's nothing to prevent the user from entering a passphrase as the password value ofr an entry, or, of course, the master password.

To summarize, I don't think the cost/benefit of this justifies the effort (but if someone wishes to create a pull request, I'm willing to reconsider...).
